package.json

```diff
@@ -13,6 +13,7 @@
   "devDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.5",
     "@nomiclabs/hardhat-waffle": "^2.0.3",
+    "@nomiclabs/hardhat-web3": "^2.0.0",
```

Comment:
This repository uses ethers/waffle.  It would make a mess to combine that with web3/truffle.

Reply:
oke I will change that